src/test/scala/cornichonTests/FeatureConfig.scala

```diff
@@ -0,0 +1,18 @@
+package cornichonTests
+import eu.timepit.refined.api.Refined
+import eu.timepit.refined.string.Url
+import pureconfig._
+import pureconfig.generic.auto._
+
+case class Config(
+  authUrl: String Refined Url,
+  clientId: String,
+  clientSecret: String,
+  projectKey: String,
+)
+
+trait FeatureConfig {
+  lazy val configuration: Config = ConfigSource.default.at("auth-test-vars").loadOrThrow[Config]
+  lazy val projectKey = configuration.projectKey
+  def authUrl:Refined[String, Url] = configuration.authUrl
+}
```

Comment:
Minor:

with git, we try to have a new line at the end: https://stackoverflow.com/questions/51957907/avoid-no-newline-at-end-of-file-in-git-diff

you can configure Intellij to add it for you:
<img width="557" alt="Screenshot 2019-09-24 at 13 35 02" src="https://user-images.githubusercontent.com/51669/65508285-29f50c80-ded0-11e9-9e6d-7969549eff3b.png">